paddle/cuda/src/hl_cuda_cublas.cc

```diff
@@ -162,6 +169,65 @@ void hl_matrix_transpose(real *A_d, real *C_d, int dimM, int dimN) {
   hl_matrix_transpose(A_d, C_d, dimM, dimN, dimN, dimM);
 }
 
+void hl_matrix_inverse(real *A_d, real *C_d, int dimN, int lda, int ldc) {
+  /* Solve Ax = I */
+  CHECK_NOTNULL(A_d);
+  CHECK_NOTNULL(C_d);
+
+  /* Step 1: Compute the LU decomposition of matrix A */
+  const unsigned int nMatrices = 1;
```

Comment:
If this api consider only calculating one matrix. Remove `nMatrices` can make the code more clear.